tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_RAP_suite_HRRR.yaml

```diff
@@ -35,3 +35,4 @@ task_get_extrn_lbcs:
   USE_USER_STAGED_EXTRN_FILES: true
 verification:
   VX_FCST_MODEL_NAME: FV3_GFS_v15p2_CONUS_25km
+  RUN_VX_DET: True
```

Comment:
Here's a really nice example of what I mean in the longer comment. To get this to work out you had to add 2 separate pieces of information so that setup.py doesn't remove all the verify_det tasks.

Reply:
@christinaholtNOAA I agree.

Reply:
@christinaholtNOAA  The `RUN_VX_[DET|ENS]` switches have been removed.